apps/ledger-live-desktop/src/newArch/WalletSync/__tests__/manageYourBackup.test.tsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * @jest-environment jsdom
+ */
+import "@testing-library/jest-dom";
+import { describe, it, expect, jest } from "@jest/globals";
+
+import React from "react";
+import { render, screen, waitFor } from "tests/testUtils";
+import WalletSyncRow from "~/renderer/screens/settings/sections/General/WalletSync";
+import { State } from "~/renderer/reducers";
+
+const WalletSyncTestApp = () => (
+  <>
+    <div id="modals"></div>
+    <WalletSyncRow />
+  </>
+);
+
+jest.mock(
+  "electron",
+  () => ({ ipcRenderer: { on: jest.fn(), send: jest.fn(), invoke: jest.fn() } }),
+  { virtual: true },
+);
+
+describe("ManageYourBackup", () => {
+  it("should open drawer and display Wallet Sync Manage flow and delete your backup", async () => {
+    const { user } = render(<WalletSyncTestApp />, {
+      overrideInitialState: (state: State) => ({
+        ...state,
+        walletSync: {
+          ...state.walletSync,
+          activated: true,
+        },
+      }),
+    });
+
+    //Open drawer
+    const button = screen.getByRole("button", { name: "Manage" });
+    await user.click(button);
+
+    const row = screen.getByTestId("walletSync-manage-backup");
+    await waitFor(() => expect(row).toBeDefined());
```

Comment:
This could be replaced by
await screen.findById
I think, to be checked with the documentation